Code/DataStructs/BitOps.cpp

```diff
@@ -785,6 +785,7 @@ template void UpdateBitVectFromBinaryText(ExplicitBitVect&, const std::string&);
 #ifdef _MSC_VER
 #include <intrin.h>
 #define __builtin_popcount __popcnt
+#define __builtin_popcountl __popcnt64
```

Comment:
The control over whether or not popcnt is used is in the global CMakeLists.txt file. I had screwed this up (so that you could not turn it off), but now it ought to work.